plugins/plugin-chart-echarts/src/Gauge/controlPanel.tsx

```diff
@@ -0,0 +1,267 @@
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { t, validateNonEmpty } from '@superset-ui/core';
+import { sharedControls, ControlPanelConfig, D3_FORMAT_OPTIONS } from '@superset-ui/chart-controls';
+import { DEFAULT_FORM_DATA } from './types';
+
+const {
+  title,
+  minVal,
+  maxVal,
+  fontSize,
+  numberFormat,
+  roundcap,
+  showSplitLine,
+  showAxisTick,
+  valueName,
+  valueFormatter,
+  valueAnimation,
+  startAngle,
+  endAngle,
+  intervals,
+  intervalColorIndices,
+} = DEFAULT_FORM_DATA;
+
+const config: ControlPanelConfig = {
+  controlPanelSections: [
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        ['groupby'],
+        ['metric'],
+        ['adhoc_filters'],
+        ['row_limit', null],
+        [
+          {
+            name: 'min_val',
+            config: {
+              ...sharedControls.metric,
+              type: 'MetricsControl',
+              label: t('Min'),
+              description: t('Minimum value on the gauge axis'),
+              default: minVal,
+              validators: [validateNonEmpty],
+            },
+          },
+        ],
+        [
+          {
+            name: 'max_val',
+            config: {
+              ...sharedControls.metric,
+              type: 'MetricsControl',
+              label: t('Max'),
+              description: t('Maximum value on the gauge axis'),
+              default: maxVal,
+              validators: [validateNonEmpty],
+            },
+          },
+        ],
+      ],
+    },
+    {
+      label: t('Chart Options'),
+      expanded: true,
+      controlSetRows: [
+        [<h1 className="section-header">{t('General')}</h1>],
+        [
+          {
+            name: 'title',
+            config: {
+              type: 'TextControl',
+              label: t('Title'),
+              renderTrigger: true,
+              default: title,
+              description: t('Chart title'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'number_format',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Number format'),
+              renderTrigger: true,
+              default: numberFormat,
+              choices: D3_FORMAT_OPTIONS,
+              description: `${t('D3 format syntax: https://github.com/d3/d3-format')} ${t(
```

Comment:
Is it required to put it like this separately?
And Second string about Label type may not be required for this chart.